server/configuration.go

```diff
@@ -166,7 +166,10 @@ func (p *Plugin) OnConfigurationChange() error {
 	}
 
 	p.setConfiguration(configuration)
-	p.metricsService.ObserveWhitelistLimit(configuration.ConnectedUsersAllowed)
+
+	if p.metricsService != nil {
+		p.metricsService.ObserveWhitelistLimit(configuration.ConnectedUsersAllowed)
+	}
```

Comment:
Should we just put this in `p.start()` which will get called both on startup and on a configuration change by virtue of `p.restart()` below?

Reply:
Good call, let me change it.

Reply:
Done